widget/table.go

```diff
@@ -90,41 +79,51 @@ func (t *Table) CreateRenderer() fyne.WidgetRenderer {
 	return r
 }
 
-// Selection will mark the specified cell (at row, col) to be marked as selected.
-func (t *Table) Selection() (row, col int) {
-	return t.selectedRow, t.selectedColumn
+// Select will mark the specified cell as selected.
+func (t *Table) Select(id TableCellID) {
+	if t.OnUnselected != nil && t.selectedCell != nil {
+		t.OnUnselected(*t.selectedCell)
```

Comment:
nit: assign callback to local var for safety
```suggestion
	if f := t.OnUnselected; f != nil && t.selectedCell != nil {
		f(*t.selectedCell)
```

Reply:
Yeah I thought about doing this too but just went with the pattern that already existed.  I'll update it.

Reply:
Thank you

Reply:
Resolved